headers/fmi3FunctionTypes.h

```diff
@@ -72,6 +71,23 @@ typedef enum {
 } fmi3InterfaceType;
 /* end::InterfaceType[] */
 
+/* tag::CoSimulationMode[] */
+typedef enum {
+    fmi3ModeCoSimulation,
+    fmi3ModeHybridCoSimulation,
+    fmi3ModeScheduledExecutionSimulation
+} fmi3CoSimulationMode;
+/* end::CoSimulationMode[] */
+
+/* tag::CoSimulationConfiguration[] */
+typedef struct{
+    fmi3Boolean intermediateVariableGet;
```

Comment:
Boolean variables should start with or contain "is", "has" or "can". So an if-clause reads like an english sentence:

```C
if (config.intermediateVariableHasBeenSet) {
  // do something
}
```

Reply:
I agree with Torsten here: Let´s find better names, and, let´s not rely on the documentation explaining what the flag is supposed to do.